tests/Command/CleanupSnapshotsCommandTest.php

```diff
@@ -1,92 +0,0 @@
-<?php
```

Comment:
IMO it makes more sense to test this as a completely functional (e2e) test, instead of mocking things.